js-pkg/server/src/run.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+const { spawnSync } = require('node:child_process')
+const { installBinary, binaryExists } = require('./get_binary')
+
+async function runBinary() {
+  try {
+    var binpath = binaryExists()
```

Comment:
I didn't realize JS would allow this? I would have thought a var would be scoped to the `try {}`, so it would be undefined later. But if it works I'm fine with it.

Reply:
var is function scope, let is block scope